docs/semgrep-code/policies.md

```diff
@@ -1,27 +1,23 @@
 ---
 slug: policies
 append_help_link: true
-title: Set rules through Policies
+title: Manage policies
 hide_title: true
-description: "The Policies page is a visual representation of the rules that Semgrep Code uses to scan code."
+description: "The Policies page is a visual representation of the rules Semgrep Code uses to scan code."
 tags:
-    - Semgrep Code
-    - Semgrep AppSec Platform
+ - Semgrep Code
+ - Semgrep AppSec Platform
 ---
 
-# Set rules through Policies
+# Manage policies
```

Comment:
Just curious why we cut the extra words? I think one cannot go without the other. When we go down this lane, the docs could be boiled down to "Manage Semgrep", and "Run Semgrep" which doesn't give a hint about how and what we're managing or running. 

IMO too much abstraction / generalization or distance from the procedure reduces some helpful keywords / context that can quickly frame a product in their mind.

Reply:
That's fair; I think I found the original title a bit confusing. Maybe something like "Managing rules through policies?" Though, I am not sure that's better, so I might just revert it until/if we come up with something else

Reply:
Hmm, if the confusion is "what do we manage/configure, the rules, the policies, or both?" We could use

- Mange rules through policies (your suggestion)

Other suggestions:

- Manage rules and policies
- Configure rules and policies

Reply:
I think "Manage rules and policies" is the clearest so will go with that. Thanks!